pkg/auth/auth_test.go

```diff
@@ -0,0 +1,74 @@
+/*This file is part of kuberpult.
```

Comment:
Also I think this is in a directory that is not tested along with the other tests in `make test` 
how can we fix this?

Reply:
It should just work?
Is this directory in the `MAKEDIRS` variable?

Reply:
No it's in `pkg/auth` but `MAKEDIRS` only has `pkg/api` 

also I just checked the test runner here on github and the `setup_test.go` is not running as well. which is in `pkg/setup`

I'll create a new PR with a new makefile in `pkg` and add that to the `MAKEDIRS`

but for now the tests are successful.